Docker/Dockerfile

```diff
@@ -0,0 +1,184 @@
+    ./configure --prefix=$DIR/grib2 && \
+    make && make install 
+
+### libpng 1.2.50
+RUN wget https://sourceforge.net/projects/libpng/files/libpng12/older-releases/1.2.50/libpng-1.2.50.tar.gz && \
+    tar -xzf libpng-1.2.50.tar.gz && \
+    cd libpng-1.2.50 && \
+    ./configure --prefix=$DIR/grib2 && \
+    make && make install
+
+# Jasper 1.900.1
+RUN wget https://www.ece.uvic.ca/~frodo/jasper/software/jasper-1.900.1.zip && \
+    unzip jasper-1.900.1.zip && \
+    cd jasper-1.900.1 && \
+    ./configure --prefix=$DIR/grib2 && \
+    make && make install
+
+
+## Install Dependencies
+### PHDF5 1.10.4
+RUN wget https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.4/src/hdf5-1.10.4.tar.gz && \
+    tar -xzf hdf5-1.10.4.tar.gz && \
+    cd hdf5-1.10.4 && \
+    ./configure --enable-fortran --enable-parallel --enable-shared --with-zlib=$DIR/grib2 --prefix=$PHDF5 && \
+    make && make install && cd ..
+
+ENV LDFLAGS="-L$PHDF5/lib -L$DIR/grib2/lib"
+RUN echo export LDFLAGS="-L$PHDF5/lib -L$DIR/grib2/lib" >> /etc/bashrc
+
+### PNetCDF 1.11.0
+RUN wget http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/pnetcdf-1.11.0.tar.gz && \
+    tar -xzf pnetcdf-1.11.0.tar.gz && \
+    cd pnetcdf-1.11.0 && \ 
+    ./configure --prefix=$PNETCDF --enable-fortran --enable-shared  && \
+    make install
+
+ENV LDFLAGS="-L$PNETCDF/lib -L$PHDF5/lib -L$DIR/grib2/lib"
+RUN echo export LDFLAGS="-L$PNETCDF/lib -L$PHDF5/lib -L$DIR/grib2/lib" >> /etc/bashrc
+
+### NetCDF 4.6.2
+RUN mkdir -p ${NETCDF} && \
+    cd ${NETCDF} && \
+    wget https://github.com/Unidata/netcdf-c/archive/v4.6.2.tar.gz && \
+    wget https://github.com/Unidata/netcdf-fortran/archive/v4.4.5.tar.gz && \
+    tar -xzf v4.6.2.tar.gz && \
+    cd netcdf-c-4.6.2 && \
+    ./configure --enable-fortran --disable-static --disable-dap --enable-shared --with-pic --enable-parallel-tests -enable-pnetcdf --enable-largefile --prefix=${NETCDF} && \
+    make && make install
+RUN cd ${NETCDF} && \
+    tar -xzf v4.4.5.tar.gz && \
+    cd netcdf-fortran-4.4.5 && \
+    ./configure  --disable-static --enable-shared --with-pic --enable-parallel-tests --prefix=${NETCDF} && \
+    make && make install
+
+#ENV NETCDF4=1
+#RUN echo export export NETCDF4=1 >> /etc/bashrc
+
+## Install WRF
+WORKDIR $MAIN_DIR
+
+ENV LDFLAGS="-lm -lnetcdff -L${PNETCDF}/lib"
+RUN echo export LDFLAGS="-lm -lnetcdff -L${PNETCDF}/lib" >> /etc/bashrc
+
+### Get v4.2.2 archive
+RUN wget https://github.com/wrf-model/WRF/archive/refs/tags/v4.2.2.tar.gz && \
+    tar -zxf v4.2.2.tar.gz && \
+    mv WRF-4.2.2/ WRF
+
+WORKDIR $WRF_SRC_ROOT_DIR
+
+### Configure
+RUN ./clean -a && \
+    echo '66\n1\n' | ./configure && \
+    sed -i -e 's/gcc/icc/' ./configure.wrf && \
+    awk -v add=" -L${DIR}/zlib-1.2.7" '/^ZLIB/ {$0=$0add} 1' configure.wrf > temp && mv temp configure.wrf
+
+### Compile
+RUN time ./compile em_real 2>&1 | tee compile_em_real.log
+
+
+#TODO: Add a check that WRF installed correctly, error otherwise
```

Comment:
Need to test that WRF `.exe` files exist (see [WRF compile tutorial](https://www2.mmm.ucar.edu/wrf/OnLineTutorial/compilation_tutorial.php#STEP4)):

* wrf.exe (model executable)
* real.exe (real data initialization)
* ndown.exe (one-way nesting)
* tc.exe (for tc bogusing--serial only)

I will plan to add this.